frontend-react/src/network/api/Organizations/Receivers.ts

```diff
@@ -0,0 +1,16 @@
+import { API } from "../NewApi";
+
+export class RSReceiver {
+    name: string = "";
+    organizationName: string = "";
+}
+export const ReceiverApi = new API(RSReceiver, "/api/settings/organizations")
+    .addEndpoint("list", "/:org/receivers", ["GET"])
+    .addEndpoint("detail", "/:org/receivers/:receiver", ["GET"]);
```

Comment:
Hmm...actually maybe add this to `OrganizationsApi` -- didn't see it before. Whoops!

Reply:
So, the reverse happened. Because we're hoping to tie calls to a definite type, it's necessary to _remove_ the `/:org/receivers` call from where it was in `OrganizationsApi` into this, so it can use the `RSReceiver` resource.